cypress/support/step_definitions/api_admin_update_registry_metadata_.js

```diff
@@ -4,55 +4,57 @@
 //    When the API admin user changes the metadata for the entity registry
 //    Then the metadata for the entity registry is updated
 
-when('the API admin user changes the metadata for the entity registry', () => {
-	cy.log('-- api_admin_update_registry_metadata_.js --')
-	
-	cy.get('@registryName').then((registryName) => {
-		cy.get('@registryAdminApiKey').then((apiKey) => {
+import {Then, When} from 'cypress-cucumber-preprocessor/steps';
 
-			let registryGetUrl = '/registry/' + registryName;
-			cy.request({
-				url: registryGetUrl,
-				method: 'GET',
-				headers: {
-					'api-key': apiKey
-				}
-			}).then((response) => {
-				expect(response.body).to.have.string('descriptionValue')
-			})
-			
-			let registryUpdateUrl = '/registry/' + registryName;
-			cy.fixture('registryTestMetadataUpdated.json').then((updatedSchema) => {
-				updatedSchema.id = registryName
-				cy.log('description: ' + updatedSchema.metadata.description)
-				cy.request({
-					url: registryUpdateUrl,
-					method: 'PUT',
-					body: updatedSchema,
-					headers: {
-						'api-key': apiKey,
-						'Content-Type': 'application/json'
-					}
-				})
-			})
-		})
-	})
-})
+When('the API admin user changes the metadata for the entity registry', () => {
+  cy.log('-- api_admin_update_registry_metadata_.js --');
 
-then('the metadata for the entity registry is updated', () => {
-	cy.get('@registryName').then((registryName) => {
-		cy.get('@registryAdminApiKey').then((apiKey) => {
-			cy.registryReady(registryName)
-			let registryGetUrl = '/registry/' + registryName;
-			cy.request({
-				url: registryGetUrl,
-				method: 'GET',
-				headers: {
-					'api-key': apiKey
-				}
-			}).then((response) => {
-				expect(response.body).to.have.string('updatedDescriptionValue')
-			})
-		})
-	})
-})
+  cy.get('@registryName').then((registryName) => {
+    cy.get('@registryAdminApiKey').then((apiKey) => {
+
+      let registryGetUrl = '/registry/' + registryName;
+      cy.request({
+        url: registryGetUrl,
+        method: 'GET',
+        headers: {
+          'api-key': apiKey,
+        },
+      }).then((response) => {
+        expect(response.body.metadata.description).to.equals('descriptionValue');
+      });
+
+      let registryUpdateUrl = '/registry/' + registryName;
+      cy.fixture('registryTestMetadataUpdated.json').then((updatedSchema) => {
+        updatedSchema.id = registryName;
+        cy.log('description: ' + updatedSchema.metadata.description);
+        cy.request({
+          url: registryUpdateUrl,
+          method: 'PUT',
+          body: updatedSchema,
+          headers: {
+            'api-key': apiKey,
+            'Content-Type': 'application/json',

```

Comment:
If text/html is saved as  const, shouldn't application/json be also saved as const?
Also,  'accept' ?

Reply:
PUT, content-type is correct